src/App.tsx

```diff
@@ -6,31 +6,68 @@ import Home from './components/Home';
 import ExamplePage from './components/ExamplePage';
 import NotFound from './components/NotFound';
 
-import { AppBar, Tabs, Tab } from '@material-ui/core';
+import { AppBar, Tabs, Tab, ThemeProvider } from '@material-ui/core';
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      // light: will be calculated from palette.primary.main,
+      main: '#6C4087',
+      // dark: will be calculated from palette.primary.main,
+    },
+    secondary: {
+      // light: will be calculated from palette.primary.main,
+      main: '#C71C5B',
+      // dark: will be calculated from palette.primary.main,
+      // contrastText: will be calculated to contrast with palette.primary.main
+    },
+    text: {
+      //primary: '',
+      //secondary: ''
+    },
+  },
+  typography: {
+    h1: {
```

Comment:
should this be applied to p tags too? We use it for everything, including google docs etc